kernel/statistics.c

```diff
@@ -0,0 +1,185 @@
+/*
```

Comment:
can we put this file in a folder, maybe paging/statistics.c? This way we know it is paging related and that it does not provide kernel statistics features in general.

Reply:
Moved.